src/integration-test/integration/ExtractionClient.test.ts

```diff
@@ -0,0 +1,62 @@
+/*---------------------------------------------------------------------------------------------
+* Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+* See LICENSE.md in the project root for license terms and full copyright notice.
+*--------------------------------------------------------------------------------------------*/
+import * as chaiAsPromised from "chai-as-promised";
+import { expect, use } from "chai";
+import { ExtractionClient, ExtractionLog, ExtractionRun, ExtractionStatus, ExtractorState, MappingCreate, MappingsClient } from "../../reporting";
```

Comment:
Unused import

Reply:
Can't find the unused import, perhaps you could specify?

Reply:
I think `ExtractorState` was not used earlier, but since you committed your local changes it is used now.